taskvine/src/manager/vine_task_groups.c

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright (C) 2022- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#include "vine_task_groups.h"
+#include "debug.h"
+#include "vine_mount.h"
+#include "vine_task.h"
+
+// create a new task group for this task based on the temp mount file
+static int vine_task_groups_create_group(struct vine_manager *q, struct vine_task *t, struct vine_mount *m)
+{
+	cctools_uuid_t uuid;
+	cctools_uuid_create(&uuid);
```

Comment:
A uuid is a relatively expensive operation needed for ensuring uniqueness across hosts and invocations.  Do we really need that here, or would an integer counter suffice?